src/transformers/models/whisper/processing_whisper.py

```diff
@@ -91,3 +96,22 @@ def decode(self, *args, **kwargs):
         the docstring of this method for more information.
         """
         return self.tokenizer.decode(*args, **kwargs)
+
+    def get_prompt_ids(self, text: str):
```

Comment:
IMO this method should live in the `tokenizer` since it's entirely related to encoding a string to tokens. We can then forward it onto the processor if we need access to it here:
```python
    def get_prompt_ids(self, text: str):
        return self.tokenizer.get_prompt_ids(text)
```

Reply:
Moved and kept the tests in the processing test file since that's what was done for `get_decoder_prompt_ids`